schemas/rule.yml

```diff
@@ -81,4 +81,31 @@
 
       example: https://en.wikipedia.org/wiki/DNS_over_TLS
 
+    - name: author
+      level: extended
+      type: keyword
+      short: Rule author
+      description: >
+        Name, organization, or pseudonym of the author or authors who created the rule used to generate this event.
+
+      example: John R. Doe, Elastic, s3cst1ltsk1n
```

Comment:
Two things about the example here.

First, please only put one value in the example. The asciidoc rendering looks funny otherwise ([see here](http://ecs_754.docs-preview.app.elstc.co/guide/en/ecs/master/ecs-rule.html)). To provide multiple different examples, for now we have to do it via the description.

Second, the alphanumeric handle looks strange. I know it's supposed to represent an online handle. Perhaps we could use a known handle like "Citizenfour", or a fictional one like "Starlord"?

![Starlord](https://media.giphy.com/media/3ofT5PXfFMpxR6KsLu/giphy.gif)

Reply:
:+1: for Star-Lord